From vijaykumar@bravegnu.org  Mon Oct 26 15:04:16 2009
From: vijaykumar@bravegnu.org
Date: Mon, 21 Sep 2009 11:23:59 +0530
Subject: Staging: poch: Include linux/types.h
To: greg@kroah.com
Cc: jayakumar.lkml@gmail.com, alexey.zaytsev@gmail.com
Message-ID: <20090921055914.577199541@bravegnu.org>

From: Vijay Kumar B. <vijaykumar@bravegnu.org>

Include linux/types.h in poch.h, so that poch.h can be included in
user application header files.

Signed-off-by: Vijay Kumar B. <vijaykumar@bravegnu.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/poch/poch.h |    3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/staging/poch/poch.h
+++ b/drivers/staging/poch/poch.h
@@ -7,6 +7,9 @@
  * include/linux for final version.
  *
  */
+
+#include <linux/types.h>
+
 struct poch_counters {
 	__u32 fifo_empty;
 	__u32 fifo_overflow;
